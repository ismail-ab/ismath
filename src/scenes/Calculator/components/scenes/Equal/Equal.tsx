import * as React from 'react';
import { CharacterButton } from '../../components';

const Equal: React.SFC = () => (
  <CharacterButton
    characterToDisplay="="
    color="secondary"
  />
);

export default Equal;