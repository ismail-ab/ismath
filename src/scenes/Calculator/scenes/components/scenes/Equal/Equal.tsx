import * as React from 'react';
import { CharacterButton } from '../../components';

interface IEqualOwnProps {
  computeConcatenatedCharacters: Function;
}

const Equal: React.SFC<IEqualOwnProps> = ({ computeConcatenatedCharacters }) => (
  <CharacterButton
    characterForCalculate="="
    characterToDisplay="="
    onClickAction={computeConcatenatedCharacters}
    color="secondary"
  />
);

export default Equal;