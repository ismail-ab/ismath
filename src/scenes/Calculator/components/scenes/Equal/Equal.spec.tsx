import { } from 'jest';
import * as React from 'react';
import { shallow } from 'enzyme';
import Equal from './Equal';

describe('Equal', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Equal />);

    expect(wrapper).toMatchSnapshot();
  });
});
