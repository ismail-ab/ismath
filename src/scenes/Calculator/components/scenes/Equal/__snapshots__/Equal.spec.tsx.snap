// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Equal renders correctly 1`] = `
<CharacterButton
  characterToDisplay="="
  color="secondary"
/>
`;
